dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/utils/DateUtilsTest.java

```diff
@@ -196,4 +196,12 @@ public void testFormat2Duration() {
 
     }
 
+    @Test
+    public void testNullDuration() {
+        // days hours minutes seconds
+        Date d1 = DateUtils.stringToDate("2020-01-20 11:00:00");
+        Date d2 = null;
+        Assert.assertEquals("",DateUtils.format2Duration(d1,d2));
```

Comment:
It's better to add a trailing space after each comma

Reply:
ok